client/src/store/index.js

```diff
@@ -1,7 +1,12 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import isLogin from './isLogin';
+
+const rootReduser = combineReducers({
+  isLogin,
+});
```

Comment:
```suggestion
import { configureStore } from '@reduxjs/toolkit';
import usersReducer from './slices/usersSlice';

const store = configureStore({
  reducer: { userReducer },
});

export default store;
```


this is our store in the correct version, no need to import anything combine users or manually combine functions,  we are using Toolkit  . it is all done under the hood and written in a simple readable code.

Reply:
done